tests/gpu-vm

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
```

Comment:
@simondeziel were you able to successfully run this test on the test flinger hardware or locally?